tests/frontend/test_engine.py

```diff
@@ -357,16 +357,14 @@ def test_engine_with_mocked_api_client_sample_job(self, monkeypatch):
         pi = 3.14
 
         with prog.context as q:
-            ops.S2gate(sqz0, 0.0) | (q[0], q[2])
-            ops.S2gate(sqz1, 0.0) | (q[1], q[3])
+            ops.S2gate(1, 0) | (q[0], q[4])
+            ops.S2gate(1, 0) | (q[1], q[5])
+            ops.S2gate(1, 0) | (q[2], q[6])
+            ops.S2gate(1, 0) | (q[3], q[7])
             ops.Rgate(phi0) | q[0]
             ops.BSgate(pi / 4, pi / 2) | (q[0], q[1])
-            ops.Rgate(phi1) | q[0]
-            ops.BSgate(pi / 4, pi / 2) | (q[0], q[1])
-            ops.Rgate(phi0) | q[2]
-            ops.BSgate(pi / 4, pi / 2) | (q[2], q[3])
-            ops.Rgate(phi1) | q[2]
-            ops.BSgate(pi / 4, pi / 2) | (q[2], q[3])
```

Comment:
Any reason this was changed? From what I can see, it shouldn't actually matter what the specific program is here

Reply:
The unitary restriction on chip2 differed from chip0. On chip2, the unitary on modes 0-3 must match the unitary on modes 4-7. So this test needed to be modified to take this into account.